server/controllers/handleLogin.js

```diff
@@ -11,19 +11,20 @@ const handleLogin = (req, res, next) => {
       if (data.rowCount === 1) {
         return getPassword(data.rows[0].email);
       } else {
-        throw customizedError("Email and password don't  match ");
+        throw customizedError("Email and password don't  match " , 400);
       }
     })
     .then((data) => {
       return compare(password, data.rows[0].password);
     })
     .then((isMatch) => {
       if (isMatch) {
-        return signPromise(email)
+        return getId(email)
       } else {
-        throw customizedError("Email and password don't  match ");
+        throw customizedError("Email and password don't  match " , 400);
       }
     })
+    .then((data)=> signPromise(data.rows[0].id))
     .then((token)=>{
         res.cookie('access_token', token, {
```

Comment:
as your token expires after 30 days, maybe it is better to add a flag to our cookies to expire at the same time.
Because what're doing now, that after 30 days your token would be expired but cookie still lives in the browser.
So it would be better to expire our token too, So after 30 days our cookies in the first place will terminate from the user browser